### DISCUSSION TITLE
Add `mode` highlighting in the `ls --long` command

### DISCUSSION THREAD

**Discussion by MrPandir** [nushell/nushell#13266] [Ideas]
Add permissions highlighting, similar to how it is done in `exa`:
<img width="1512" src="https://github.com/nushell/nushell/assets/137798474/e2b2e582-9ef1-49be-9534-b895f7c7fee5">

**Comment by fdncred** [comment]
I think it would look cool but other than that I'm not sure what benefit it would have. I'm guessing we'd accept a PR that mimics the eza/exa syntax-highlighting of permissions.
